database.js

```diff
@@ -0,0 +1,51 @@
+let express = require("express");
+let { Pool } = require("pg");
+let env = require("E:/starter2/env.json");
+const argon2 = require('argon2');
+
+let hostname = "localhost";
+let port = 3000;
+let app = express();
+
+app.use(express.json());
+
+let pool = new Pool(env);
+pool.connect().then(() => {
+    console.log("Connected to database");
+});
+
+app.get('/login', async (req, res) => {
```

Comment:
Great work using async/await, it's probably good we start getting ride of nested then/catches to improve clarity